mappings/net/minecraft/block/ChestBlock.mapping

```diff
@@ -21,6 +21,14 @@ CLASS net/minecraft/class_118 net/minecraft/block/ChestBlock
 	METHOD method_16648 getOpenStat ()Lnet/minecraft/class_4472;
 	METHOD method_16649 getFacing (Lnet/minecraft/class_2232;)Lnet/minecraft/class_1354;
 		ARG 0 state
+	METHOD method_8701 (Lnet/minecraft/class_1150;Lnet/minecraft/class_2552;Lnet/minecraft/class_2232;)Lnet/minecraft/class_2232;
```

Comment:
is this right?

Reply:
I missed this, what is being asked if it's right?